src/reporters/lang/en.js

```diff
@@ -116,6 +116,8 @@ const messages = {
   multiplePackagesCantUnpackInSameDestination:
     'Pattern $0 is trying to unpack in the same destination $1 as pattern $2. This could result in a non deterministic behavior, skipping.',
   incorrectLockfileEntry: 'Lockfile has incorrect entry for $0. Ignoring it.',
+  cwdInYarnrcWarning:
+    'There was a --cwd flag in a .yarnrc file relative to this cwd. .yarnrc files relative to the --cwd have been ignored.',
```

Comment:
I don't see this message being referenced anywhere - @kittens is it still used or can we remove it?